apps/web/src/components/ConfirmEventModal/ConfirmEventModal.tsx

```diff
@@ -4,7 +4,7 @@ import Alert from 'react-bootstrap/Alert';
 import Button from 'react-bootstrap/Button';
 import Select from 'react-select';
 
-import HeatMapData from '../../interfaces/HeatMapData';
+import { HeatMapData } from '../../interfaces/index';
```

Comment:
Nit: don't need to import the index. Everytime you import from dir it looks for index.js or index.ts first

Reply:
I saw that in another file, wasn't sure if I should still explicitly state it. I'll remove them, thanks